src/mame/f256/f256.h

```diff
@@ -0,0 +1,205 @@
+// license:BSD-3-Clause
+// copyright-holders:Daniel Tremblay
+#ifndef MAME_F256_F256_H
+#define MAME_F256_F256_H
+
+#pragma once
+
+#include "machine/ram.h"
+#include "screen.h"
+#include "machine/bq4847.h"
+#include "machine/6522via.h"
+#include "sound/sn76496.h"
+#include "sound/ymopl.h"
+#include "sound/mos6581.h"
+#include "utf8.h"
+#include "tiny_vicky.h"
+#include "speaker.h"
+#include "machine/spi_sdcard.h"
+// PS/2 mouse and keyboard
+#include "bus/pc_kbd/pc_kbdc.h"
+#include "bus/pc_kbd/hle_mouse.h"
+#include "debug/debugcon.h"
+#include "debugger.h"
+//#include "bus/cbmiec/cbmiec.h"
+//#include "bus/cbmiec/c1581.h"
+#include "machine/ins8250.h"
+
+#define MASTER_CLOCK        (XTAL(25'175'000))
+#define MUSIC_CLOCK         (XTAL(14'318'181))
+#define MAINCPU_TAG                 "maincpu"
```

Comment:
In general, using finders should remove the need for these `TAG` preprocessor macros.